lib/dialects/sqlite3/query/compiler.js

```diff
@@ -2,20 +2,17 @@
 
 const QueryCompiler = require('../../../query/compiler');
 
-const {
-  constant,
-  each,
-  isEmpty,
-  isString,
-  noop,
-  reduce,
-  identity,
-} = require('lodash');
+const constant = require('lodash/constant');
+const each = require('lodash/each');
+const identity = require('lodash/identity');
+const isEmpty = require('lodash/isEmpty');
+const isString = require('lodash/isString');
+const noop = require('lodash/noop');
```

Comment:
there is already an internal implementation of noop that can be used: lib\util\noop.js